topology/ANL/ANL ASC/ANLASC.yaml

```diff
@@ -0,0 +1,243 @@
+Disable: false
+GridType: OSG Production Resource
+GroupID: 283
+Resources:
+  ANL-ATLAS-GRIDFTP1:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary: Doug Benjamin
+      Security Contact:
+        Primary: Doug Benjamin
+      Submitter Contact:
+        Primary: Doug Benjamin
+    Disable: false
+    FQDN: atlasgridftp.hep.anl.gov
+    FQDNAliases:
+    - atlasgridftp1.hep.anl.gov
+    ID: 437
+    Services:
+      GridFtp:
+        Description: GridFtp Storage Element
+        Details:
+          hidden: false
+          uri_override: atlas8.hep.anl.gov:2811
+        ID: 5
+    VOOwnership:
+      ATLAS: 100
+    WLCGInformation:
+      APELNormalFactor: 0
+      AccountingName: null
+      HEPSPEC: 0
+      InteropAccounting: false
+      InteropBDII: true
+      InteropMonitoring: false
+      KSI2KMax: 0
+      KSI2KMin: 0
+      LDAPURL: ldap://is.grid.iu.edu:2180/mds-vo-name=ANLASC,o=grid
+      StorageCapacityMax: 19
+      StorageCapacityMin: 0.5
+      TapeCapacity: 0
+  ANL-ATLAS-SE:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary: Doug Benjamin
+      Security Contact:
+        Primary: Doug Benjamin
+      Submitter Contact:
+        Primary: Doug Benjamin
+    Disable: false
+    FQDN: atlasgridftp1.hep.anl.gov
+    ID: 747
+    Services:
+      GridFtp:
+        Description: GridFtp Storage Element
+        Details:
+          hidden: false
+          uri_override: null
+        ID: 5
+    VOOwnership:
+      ATLAS: 100
+  ANLASC Squid:
+    Active: false
+    ContactLists:
+      Administrative Contact:
+        Primary: Doug Benjamin
+      Security Contact:
+        Primary: Doug Benjamin
+      Submitter Contact:
+        Primary: Doug Benjamin
+    Disable: false
+    FQDN: squid.hep.anl.gov
+    ID: 595
+    Services:
+      Squid:
+        Description: Generic squid service
+        Details: null
```

Comment:
Are `null` details a default? The previous gridftp resources in this rg had `uri_override` and `hidden`